src/guide/scaling-up/testing.md

```diff
@@ -1,32 +1,75 @@
 # Testing
 
-## Overview
+## Why Test?
+
+Automated tests help you and your team build complex Vue applications quickly and confidently by preventing regressions and encouraging you to break apart your application into testable functions, modules, classes, and components. As with any application, your new Vue app can break in many ways, and it's important that you can catch these issues and fix them before releasing.
 
-When it comes to building reliable applications, automated tests can play a critical role in an individual or team's ability to build new features, refactor code, and fix bugs with confidence.
+In this guide, we'll cover basic terminology and provide our recommendations on which tools to choose for your Vue 3 application.
 
-In a Vue application, there are different types of concerns that can be covered by automated tests:
+There is one Vue-specific section covering composables. See [Testing Composables](#testing-composables) below for more details.
 
-- **Logic**: verify non-UI rendering logic is implemented correctly. This typically involves application-wide business logic (e.g. state management stores) and utility functions.
+:::tip Start testing early!
```

Comment:
**suggestion**: we try to avoid "special" blocks like tips or warnings as much as possible as we consider them destructive for the reading flow. Is there any chance we could omit the `tip` part while keeping the valuable suggestion?